supervisor/hardware/manager.py

```diff
@@ -101,10 +101,22 @@ def _import_devices(self) -> None:
         self._devices.clear()
 
         # Exctract all devices
-        for device in self._udev.list_devices():
+        for device in self._udev.list_devices():            
             # Skip devices without mapping
-            if not device.device_node or self.helper.hide_virtual_device(device):
-                continue
+            try:
+              if self.helper.hide_virtual_device(device):  
+                  continue
+            except:
+              _LOGGER.exception("Exception:")
+              continue
+
+            try:
+              if not device.device_node:
+                  continue
+            except:
+              _LOGGER.exception("Exception:")
+              continue
+                
```

Comment:
Let's use a single try/except here and catch the specific exception here as suggested by @pvizeli. Also add a note that this is something pending upstream, e.g.

```suggestion
                if not device.device_node or self.helper.hide_virtual_device(device):
                    continue
            except UnicodeDecodeError as err:
                # Some udev properties have an unkown/different encoding. This is a general
                # problem with pyudev, see https://github.com/pyudev/pyudev/pull/230
                _LOGGER.warning("Ignoring udev device due to error: %s", err)
                continue
```